### PR TITLE
Experience of buying a land for summerhouse in a national park

### PR DESCRIPTION
**Pull request by sigrks**
To get an official construction permit in a national park - sometimes you can, sometimes you cannot. Wat are criteria to check against in order to be sure that you are going to receive desired construction permit? This presentation provides key aspects and know-how gained on what to evaluate while looking at some land to buy in a national park, including the process on how to go there.

### CODE REVIEW

**Comment by Pawka** [comment]
@sigrks Pridėk šiokį tokį aprašymą į PR'o descrption lauką. Ačiū.

File: 404.txt [hunk 1]
```diff
@@ -28,6 +28,7 @@ interested in travel stories, life journeys, startups, commercial sh*t,
   PR1   Intro: Camp Not Found ........................................... @pawka
   PR108 Myths and truths about hiking gear and prep: what I learned… . @EgleSilk
   PR109 Photography hacks for begginers to make better pictures ....... @dainora
+  PR110 Experience of buying a land for summerhouse in a national park. @sigrks
```
Comment (sigrks) [inline 404.txt:31]: To get an official construction permit in a national park - sometimes you can, sometimes you cannot. Wat are criteria to check against in order to be sure that you are going to receive desired construction permit? This presentation provides key aspects and know-how gained on what to evaluate while looking at some land to buy in a national park, including the process on how to go there.

**Comment by sigrks** [comment]
Pridėjau, jeigu teisingai supratau, kur pridėti. Copypastinu ir čia "To get an official construction permit in a national park - sometimes you can, sometimes you cannot. Wat are criteria to check against in order to be sure that you are going to receive desired construction permit? This presentation provides key aspects and know-how gained on what to evaluate while looking at some land to buy in a national park, including the process on how to go there. "

**Comment by Pawka** [comment]
Ačiū!

**Pull request merged**